app/src/androidTest/java/com/github/sdp/mediato/ProfileFragmentTest.java

```diff
@@ -111,6 +117,63 @@ public void testAddMediaButton() {
     collectionRecyclerView.check(matches(hasItemCount(initialItemCount + 1)));
   }
 
+  // Test the initial state of the list of collections after profile creation
+  @Test
+  public void testInitialOuterRecyclerViewState() {
+    ViewInteraction outerRecyclerView = onView(withId(R.id.collection_list_recycler_view));
+
+    // Check that the outer RecyclerView is displayed
+    outerRecyclerView.check(matches(isDisplayed()));
+
+    // Check that the outer RecyclerView initially has one item (default collection)
+    outerRecyclerView.check(matches(hasItemCount(1)));
+  }
+
+  // Check that a new collection has been added to the outer RecyclerView if the user chooses a valid collection name
+  @Test
+  public void testAddValidCollection() throws UiObjectNotFoundException {
+    ViewInteraction outerRecyclerView = onView(withId(R.id.collection_list_recycler_view));

```

Comment:
Is there a way to store these ViewInteractions as an attribute to the class or does it not work ?

Reply:
Good point I changed it.